src/NuGet.Clients/NuGet.VisualStudio.Client/.vsixinclude

```diff
@@ -30,6 +30,7 @@ NuGet.ProjectModel.dll
 NuGet.Protocol.dll
 NuGet.Resolver.dll
 NuGet.Versioning.dll
+Microsoft.DataAI.NuGetRecommender.Contracts.dll
```

Comment:
Isn't this assembly supposed to be embeddable?
Might need to add it embedinterops.targets and move this to vsixexlude.

Reply:
I tried adding it to embedinterops.targets and moving this from .vsixinclude to .vsixignore but I get errors :
Cannot embed interop types from assembly 'Microsoft.DataAI.NuGetRecommender.Contracts, Version=2.0.0.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a' because it is missing the 'System.Runtime.InteropServices.GuidAttribute' attribute.
Cannot embed interop types from assembly 'Microsoft.DataAI.NuGetRecommender.Contracts, Version=2.0.0.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a' because it is missing either the 'System.Runtime.InteropServices.ImportedFromTypeLibAttribute' attribute or the 'System.Runtime.InteropServices.PrimaryInteropAssemblyAttribute' attribute.
Am I missing another step?

Reply:
Looks like the assembly was not built as embeddable. cc @bmajik 

Is the assembly embeddable or?

Reply:
Fixed now.